### ISSUE TITLE
I don't think the module is working correctly in the current jamovi build (2.3.0)

### ISSUE THREAD

**Issue by scroono** [lucasjfriesen/jamoviPsychoPDA#22]
Hi there - the module doesn't seem to be able to accept a number as the positive class and even when you mutate the dataset so that the default settings for the program correctly identify a case, the output appears incorrect - the ROC curve is inverted and youden values are less than 1, for example.

**Comment by scroono** [comment]
....maybe it was me as the class can be selected by typing the text name in the 'positive class' input box but even then the output is not specifying any information in relation to optimal cutpoints in the advanced options....

**Comment by lucasjfriesen** [comment]
That's not ideal. I'll see about checking on this. Thanks for the note.

**Comment by jonathon-love** [comment]
no, i'm not aware of anything that would impact it, but let me know if you want a hand understanding any error messages, etc.

**Comment by lucasjfriesen** [comment]
> ....maybe it was me as the class can be selected by typing the text name in the 'positive class' input box but even then the output is not specifying any information in relation to optimal cutpoints in the advanced options....

Could you send me an email with the saved .jmv file? It will be easier to debug seeing the data. If you can, send it to: lucasjfriesen+jmv@gmail.com
